docs/UserGuide.md

```diff
@@ -135,29 +141,77 @@ Examples:
 
 Returns the book which is loaned by some user.
 
-Format: `return <user index> <book index>`
+Format: `return <book index>`
 
-* Returns the book which is loaned by some user at their respective specified `INDEXES`.
-* The indexes refer to the index number shown in the displayed user and book list respectively.
-* The indexes **must be a positive integer** 1, 2, 3, …​
+* Returns the book which is loaned by some user at the book's specified `INDEX`.
+* The index refers to the index number shown in the displayed book list respectively.
+* The index **must be a positive integer** 1, 2, 3, …​
 
 Examples:
-* `return 3 2` returns book number two which is loaned by user number three.
+* `return 2` returns book number two (which has been previously loaned).
 
 ### Loaning a book : `loan`
 
-Loans a book to some user
+Loans a book to some user, which has a due date.
 
-Format: `loan <user index> <book index>`
+Format: `loan <user index> <book index>` or `loan <user index> <book index> <due date>`
```

Comment:
Perhaps the format of the command should be standardised with the other command formats. So something like:
`loan USER_INDEX [DUE_DATE]`